docs/guides/chip_tool_guide.md

```diff
@@ -1013,3 +1021,210 @@ In this command:
 ```
 $ ./chip-tool onoff toggle 1 1 --commissioner-name beta
 ```
+
+<hr>
+
+### Subscribing to events or attributes
+
+Subscribing to an event or an attribute lets you mirror the state of the event
+or the attribute as it changes in the Matter network. The list of events or
+attributes you can subscribe to depends on the chosen cluster.
+
+You can have more than one subscription at any given time and subscribe to more
+than one attribute or event within one subscription (those attributes or events
+can come from different clusters). However, you cannot subscribe to both
+attributes and events as part of a single subscription. In other words, each
+subscription must be dedicated exclusively to either attributes or events.
+
+For more information about subscriptions, see the Matter specification at
+chapter 8, section 5 (Subscribe Interaction).
+
+> **Note:** The subscription behavior will be different if you set the
+> subscription to be sent with the parameter `isUrgent` set to `True`. See the
+> Matter specification for more information.
+
+#### Subscribing to an attribute
+
+The following procedure will use the `doorlock` cluster as an example. Complete
+the following steps:
+
+1. Start the CHIP Tool in [interactive mode](#interactive-mode) by running the
+   following command:
+
+    ```
+    $ ./chiptool interactive start
+    ```
+
+1. Run the following command to display all the available attributes you can
+   subscribe to for the given `<cluster-name>`:
+
+    ```
+    $ ./chiptool <cluster-name> subscribe
```

Comment:
You would do the subscribe from inside the interactive shell, no?

Reply:
```suggestion
    >>> <cluster-name> subscribe
```